libs/datasets/latest_values_dataset.py

```diff
@@ -191,7 +191,7 @@ def get_data_for_fips(self, fips) -> dict:
 
         Returns: Dictionary with all data for a given fips code.
         """
-        row = self.data[self.data[self.Fields.FIPS] == fips]
+        row = self.data[self.data[self.Fields.FIPS] == fips].where(pd.notnull(self.data), None)
```

Comment:
Would be helpful to add this as either as a comment/docstring or maybe have an option that defaults to true (`replace_nan_with_none` or something).

Mostly since the resulting rows of the option are changed to object, that may not always be what we want.